run_dir/design/flowcell.html

```diff
@@ -67,8 +79,27 @@
                         title="Treshold: {{ treshold }} million">{% for thousand in lane.get('clustersnb', '').split(',') %}<span class="thousand_group">{{ thousand }}</span>{% end %}
                     </td>
                     <th>%&nbsp;bases&nbsp;>&nbsp;Q30:</th>
-                    <td class="text-left {% if lane.get('overthirty', 0) < 30 %} danger {% elif lane.get('overthirty', 0) < 80 %}
-                      warning {% else %} success {% end %}">{{ lane.get('overthirty', 0) }}
+                    {% if flowcell.get('run_mode', '') == 'HiSeq X' %}
```

Comment:
Is this not something that should go in the Python back end instead of in a template?